src/components/AutoComplete/AutoComplete.js

```diff
@@ -95,7 +95,9 @@ export default class AutoComplete extends Component {
 
   render() {
     const { value } = this.state;
-    const { id, source } = this.props;
+    const { source } = this.props;
```

Comment:
Is it relevant?

Reply:
I'm not sure I follow the question, sorry 😕.  Is what relevant?  I needed to peel `id` off from this destructuring so I could turn it into a `let` and alter it.

Is that what you're asking?

Reply:
Sorry, moved to the right place :)